### ISSUE TITLE
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

### ISSUE THREAD

**Issue by pamuro** [skavanagh/KeyBox#220]
Hi.

I have carefully followed the instructions given in this forum and the Jetty links to add a ssl certificate from LetsEncrypt to keybox. I still have an error in Chrome: "ERR_SSL_VERSION_OR_CIPHER_MISMATCH" 

I have certified my certificate in this way:

> Keytool -import -alias jetty -file /root/nagiosxi.xmartic.com/fullchain.pem -keystore jetty / etc / keystore -trustcacerts

> Java -cp jetty / lib / jetty-util-9.4.2.v20170220.jar org.eclipse.jetty.util.security.Password S4rum4n01

And then edit the files to add the OBF

I do not know what I'm doing wrong.

I will appreciate your help.

**Comment by skavanagh** [comment]
It sounds like it's using an older protocol (maybe SSLv3). You can check by doing
```
java -jar jetty/start.jar jetty.server.dumpAfterStart=true
```  
you should see something like
```
 +- Protocol Selections
 |   |       |   +- Enabled (size=3)
 |   |       |   |   +- TLSv1
 |   |       |   |   +- TLSv1.1
 |   |       |   |   +- TLSv1.2
 |   |       |   +- Disabled (size=2)
 |   |       |       +- SSLv2Hello - ConfigExcluded:'SSLv2Hello'
 |   |       |       +- SSLv3 - JreDisabled:java.security, ConfigExcluded:'SSLv3'
```
Also, you may have an extra step or two with non-self signed certs.  Someone posted a guide here https://github.com/skavanagh/KeyBox/issues/130#issuecomment-231213459

**Comment by pamuro** [comment]
Thanks, but that was not the problem.

I followed the instructions of the document but at the moment of loading the certificate "jetty.pkcs12", the command did not respond. Studying something else I replaced the document line with "keytool -importkeystore -srckeystore /jetty.pkcs12 -srcstoretype PKCS12 -destkeystore keystore", which worked without problems and allowed me to continue successfully.

Thank you !!!!, we can close this thread
